bwakit/Makefile

```diff
@@ -7,7 +7,8 @@ build: ${build_tool}
 
 ${build_tool}: Dockerfile
 	docker build -t ${name}:${tag} .
-	docker tag -f ${name}:${tag} ${name}:latest
+	-docker rmi ${name}:latest
```

Comment:
I think there is a version of docker where you cannot force tag an image. So, @hannes-ucsc suggested removing the "old" latest image before tagging the new image.